resources/js/old components/ToneGenPts.js

```diff
@@ -0,0 +1,79 @@
+// import React from 'react';
+import Tone from 'tone';
+import pts, {
+    Pt,
+    // Group, 
+    // Line, 
+    // Create, 
+    Sound,
+    // Triangle, 
+    // Const, 
+    // Geom 
+    // this.synth
+
+
+} from 'pts';
+import { PtsCanvas } from "react-pts-canvas";
+
+export default class ToneGenPts extends PtsCanvas {
+    constructor(props) {
+        super(props);
+        this.index = new Pt(-1, -1);
+        this.synth = new Tone.Synth().toMaster();
+        this.sound = Sound.from(this.synth, this.synth.context).analyze(128);
+        // this.getFromDB = this.getFromDB.bind(this);
+        this.color = "ffffff";
+    }
+
+
+
+    animate(time) {
```

Comment:
As in some of the other files, I'm impressed by what you're doing here. Nice work!